### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by danielgtaylor**
This splits the unmodified code from Fury.js into its own package for parsing Swagger 2.0.

In the future we will need to clean up the code and add a few missing features.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Fury Swagger 2.0 Adapter
+
+[![Build Status](https://img.shields.io/travis/apiaryio/fury-adapter-swagger.svg)](https://travis-ci.org/apiaryio/fury-adapter-swagger) [![Coverage Status](https://img.shields.io/coveralls/apiaryio/fury-adapter-swagger.svg)](https://coveralls.io/r/apiaryio/fury-adapter-swagger) [![NPM version](https://img.shields.io/npm/v/fury-adapter-swagger.svg)](https://www.npmjs.org/package/fury-adapter-swagger) [![License](https://img.shields.io/npm/l/fury-adapter-swagger.svg)](https://www.npmjs.org/package/fury-adapter-swagger)
+
+This adapter provides support for parsing [API Blueprint](https://apiblueprint.org/) in [Fury.js](https://github.com/apiaryio/fury.js) using the Node binding [Protagonist](https://github.com/apiaryio/protagonist) of [Drafter](https://github.com/apiaryio/drafter).
```
Comment (kylef) [inline README.md]: This is wrong, this is fur swagger and not API Blueprint.

Comment (kylef) [inline README.md]: Would also be good to mention it is one way, and only for swagger parsing and not writing.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+language: node_js
+sudo: false
+node_js:
+#- '2'
+#- '1'
+#- '0.12'
```
Comment (pksunkara) [inline .travis.yml]: Is this because of robot skirt in fury? And what happened to 4 and 3 too?

Comment (danielgtaylor) [inline .travis.yml]: Yeah I'm going to update these once a version of Fury is released without the legacy stuff.

**Pull request merged**